src/java/org/apache/cassandra/repair/autorepair/AutoRepairUtils.java

```diff
@@ -538,8 +809,8 @@ else if (!hostIdsInCurrentRing.contains(nodeHistory.hostId))
                 }
             }
 
-            //get current repair status
-            CurrentRepairStatus currentRepairStatus = getCurrentRepairStatus(repairType, autoRepairHistories);
+            // get updated current repair status
+            CurrentRepairStatus currentRepairStatus = getCurrentRepairStatus(repairType, getAutoRepairHistory(repairType), myId);
             if (currentRepairStatus != null)
             {
                 logger.info("Latest repair status {}", currentRepairStatus);
```

Comment:
Wondering if we should consider changing this to debug?  It's a bit noisy otherwise as it logs the repair history for each node.

Reply:
Ok, please go ahead.